tests/acceptance/testall

```diff
@@ -963,13 +971,14 @@ then
             ALL_TESTS="$ALL_TESTS${ALL_TESTS:+ }$test"
         elif [ -d $test ]
         then
-            ALL_TESTS="$ALL_TESTS${ALL_TESTS:+ }$(find "$test" -name workdir -prune -o -name '*.cf' -print | sort)"
+            ALL_TESTS="$ALL_TESTS${ALL_TESTS:+ }$(find "$test" -name workdir -prune -or -name '*.cf' -and -not -name '*.srv.cf' -print | sort)"
         else
             echo "Unable to open test file/directory: $test"
         fi
     done
 else
-    ALL_TESTS="$ALL_TESTS${ALL_TESTS:+ }$(find . -name workdir -prune -o -name '*.cf' -print | sort)"
+    # Skip workdir, find all files matching *.cf but not *.srv.cf
+    ALL_TESTS="$ALL_TESTS${ALL_TESTS:+ }$(find . -name workdir -prune -or -name '*.cf' -and -not -name '*.srv.cf' -print | sort)"
```

Comment:
NAK. All `*.cf` files are tests, and then we have a bunch of other files that are auxiliary files, like `.srv` and `.cf.sub`. Lets not complicate the logic even further, and mix the preexisting coding style with no real benefit.